source/common/config/api_type_oracle.cc

```diff
@@ -5,6 +5,15 @@
 namespace Envoy {
 namespace Config {
 
+// To exclude rejecting below v2 protos.
+std::set<absl::string_view> exclude_v2_protos = {
```

Comment:
This is a global static non-POD, so needs to be wrapped in something like `CONSTRUCT_ON_FIRST_USE`.

Reply:
Ok. Just done in my local env. shall push code ASAP for review